src/components/molecules/Inbox.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <h1 class="font-heading text-4xl p-2">Inbox</h1>
+  <inbox-item tabindex="0" />
```

Comment:
Each InboxItem could have a cursor-pointer to promote the fact that they are clickable.

Reply:
This is a really good idea. Adds a lot of personality and makes for a more intuitive experience